api/v1/dspipeline_types.go

```diff
@@ -55,6 +55,19 @@ type DSPASpec struct {
 	*WorkflowController `json:"workflowController,omitempty"`
 }
 
+type ManagedPipelineOptions struct {
+	// Include managed pipelines with the deployment of this DSP API Server. Default: nil
```

Comment:
Could you please use the documentation from https://github.com/opendatahub-io/opendatahub-operator/commit/fe3c5714b047b74851e722977d0ac42563e2559f#diff-0c83bf3dcfa0274534f5dadb2d31caf629f420b0691ed1ec46eb9f78112ff01eR8-R23 since that was reviewed by the Documentation team?

Reply:
ah right, about this:  thinking aloud, i wonder if we should wholesale use the operator API rather than re-implementing it here in out own.  The benefits would be removing duplication of identical APIs and allowing updates in-sync with the Platform Operator, but at the cost of flexibility (can a DSPA support a ManagedPipeline not specified by the PlatformOperator-side API?) and speed-of-changes (need to wait on platform to implement/release)

This all may be moot depending on the future of the platform-scope enablement, so for now i suppose the best course of action is to just manually copy it wholesale (except for the bits that reference other PlatOperator APIs)

Reply:
the operator is not using this api, so it will only exist here